src/Kunstmaan/TranslatorBundle/AdminList/TranslationAdminListConfigurator.php

```diff
@@ -64,6 +71,30 @@ public function buildFields()
         $this->addField('status', 'kuma_translator.adminlist.header.status', true);
     }
 
+    public function getExportFields()
+    {
+        if (empty($this->exportFields)) {
+            $this->addExportField('domain', 'kuma_translator.adminlist.header.domain');
+            $this->addExportField('keyword', 'kuma_translator.adminlist.header.keyword');
+
+            $this->locales = array_unique($this->locales);
+
+            // Field building hack...
+            foreach ($this->locales as $locale) {
+                $this->addExportField($locale, strtoupper($locale));
+            }
+        }
+
+        return $this->exportFields;
+    }
+
+    public function addExportField($name, $header)
+    {
+        $this->exportFields[] = new Field($name, $header, false, null, null);
```

Comment:
The `false, null, null` arguments can be removed as these are the default values